### ISSUE TITLE
Witches Oven recipes shown incorrectly in NEI

### ISSUE THREAD

**Issue by elifoster** [elifoster/WitcheryPatch#5]
Some recipes are shown backwards:
* Steak/Raw Beef
* Meaty Stew/Raw Meaty Stew
* Cooked Mutton/Raw Mutton (Pam's)

**Comment by DarianLStephens** [comment]
Huh, and here I thought the Witches Oven could be used to unsmelt thing somehow.

**Comment by elifoster** [comment]
Yep, I also thought this. I figured it was just a "magic" thing lol
